packages/core/src/components/advanced-select/advanced-select.interfaces.ts

```diff
@@ -0,0 +1,15 @@
+import { AdvancedSelectGroupRedirect, AdvancedSelectOption } from "./advanced-select.models";
```

Comment:
`.interfaces.ts` en `.models.ts` zijn een foutje: De ene keer noemen we het models, en de andere keer interfaces. Ik heb voorkeur voor `.models.ts`, omdat models zowel interfaces als types toestaat.

Wil je deze file mergen met `advanced-select.models.ts` ?

@iterox fyi, wat denk jij?